src/core/logical_layers/renderers/stylesConfigs/mcda/calculations/index.ts

```diff
@@ -6,11 +6,20 @@ import type { MCDAConfig, TransformationFunction } from '../types';
 const equalSentiments = (a: Array<string>, b: Array<string>) =>
   a.length === b.length && a.every((x, i) => x === b[i]);
 
+// x - hex value
+// min, max - current range min and max
+// datasetMin - minimal value in dataset for indicator, constant value
```

Comment:
This comment related to `transform `function, maybe place it before it, not whole interface

Reply:
Moved the comment, also changed it a bit to make more informative